src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
Yes that's probably easy to clean up at https://github.com/brunch/brunch/blob/master/src/watch.coffee#L585-L586

Either just wrap in a `try` or specifically exclude `-`